ds701_book/jupyter_notebooks/10-Low-Rank-and-SVD.ipynb

```diff
@@ -29,6 +29,96 @@
         "::::\n",
```

Comment:
<p>Hmm... I wonder why this is showing as a change from your PR when it is already merged?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/DS701-Course-Notes/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Not sure. I think that it might be because this branch was merged into main before these changes. No I am trying to merge the latest changes back in. Are you opposed to merging this in? I can always try to interactively rebase this, but it is going to take me some time to do that.</p>

Reply:
Oh. It seems NBReview is confused in this case. The PR diff page shows just the rank index changes as you say ,but NBReview shows a bunch of changes that I made previously and already merged. We can disregard this comment.

Reply:
Oh. I see. Not that's fine.